customer_service/src/Controllers/CustomerController.php

```diff
@@ -0,0 +1,312 @@
+        $url = "localhost:81/broomy_demo/customersessions";
+        $http_client = new Client();
+        $response = $http_client->request($request_method, $url, ['headers' => $headers]);
+        $decodedResponse = json_decode($response->getBody(), true);
+
+        if (!is_array($decodedResponse) || $decodedResponse['status'] === 'failed') {
+            $responseData->addValue("authentication", "failed");
+            $responseData->addValue("href", "app/customers/search");
+            echo $responseBuilder->setName("searchcustomers")
+            ->setStatus("failed")
+            ->addClientResponse($responseData)
+            ->build();
+            // http_response_code(500);
+            return;
+        }
+
+        if ($customers === null) {
+            $responseData->addValue("href", "app/customers/search");
+            echo $responseBuilder->setName("searchcustomers")
+            ->setStatus("failed")
+            ->addClientResponse($responseData)
+            ->build();
+            // http_response_code(500);
+            return;
+        }
+
+        foreach ($customers as $customer) {
+            $responseData = new JsonClientResponseData();
+
+            $responseData->addValue("customerid", $customer->getCustomerId());
+            $responseData->addValue("firstname", $customer->getFirstName());
+            // $responseData->addValue("middlename",$customer->getMiddleName());
+            $responseData->addValue("lastname", $customer->getLastName());
+            $responseData->addValue("gender", $customer->getGender());
+            $responseData->addValue("country", $customer->getNationality());
+            $responseData->addValue("dateofbirth", $customer->getDateOfBirth());
+            $responseData->addValue("phonenumber", $customer->getPhoneNumber());
+            $responseData->addValue("email", $customer->getEmail());
+            // $responseData->addValue("alias",$customer->getAlias());
+            $responseData->addValue("profilepicture", $customer->getImage());
+            $responseData->addValue("datesignup", $customer->getDateSignup());
+            $responseData->addValue("accountstatus", $customer->getStatus());
+            $responseArray->addResponse($responseData);
+        }
+
+        echo $responseBuilder->setName("searchcustomers")
+        ->setStatus("successful")
+        ->addClientResponse($responseArray)
+        ->build();
+
+        http_response_code(200);
+    }
+    
+    private function resetCustomerPassword($email, $phoneNumber)
+    {
+        //TODO:VALIDATE INPUTS
+        $this->customer = $this->customerService
+        ->resetCustomerPassword($email, $phoneNumber);
+        
+        $responseData = new JsonClientResponseData();
+        $responseBuilder = new JsonResponseBuilder();
+
+        if ($this->customer === null) {
+            $responseData->addValue("href", "app/customers");
+            echo $responseBuilder->setName("resetcustomerpassword")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$responseBuilder'.](https://app.codacy.com/gh/BuildForSDG/Team-064-Backend/pullRequest?prid=5616370)